package.json

```diff
@@ -1,18 +1,29 @@
 {
-	"license": "MIT",
-	"dependencies": {
-		"@typescript-eslint/eslint-plugin": "^5.51.0",
-		"@typescript-eslint/parser": "^5.51.0",
-		"eslint": "^8.35.0",
-		"eslint-config-prettier": "^8.6.0",
-		"eslint-plugin-prettier": "^4.2.1",
-		"prettier": "^2.8.4",
-		"typescript": "^4.9.5",
-		"vite": "^4.1.4",
-		"vitest": "^0.29.2"
-	},
-	"scripts": {
-		"format": "prettier --write .eslintrc.cjs && eslint --quiet --fix --ext .cjs,.ts .eslintrc.cjs",
-		"test": "pnpm -r test"
-	}
+  "author": "Hugging Face",
+  "repository": "github:huggingface/huggingface.js",
+  "scripts": {
+    "build": "pnpm --recursive build",
+    "test": "pnpm --recursive test",
```

Comment:
```suggestion
```

Sorry, no, but there's a good reason why not :)

In the CI, we do `pnpm --filter "...[origin/main]" test`. We do this to only run tests on packages that were updated.

By adding this here, since the root's files are always changed, it will always run ALL tests, when we want to only run _some_ tests.